<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
  <head>
    <title>Introduction to BioinformaticsðŸ§¬ðŸ’»</title>
    <meta charset="utf-8" />
    <meta name="author" content="Martin Skarzynski" />
    <meta name="date" content="2019-11-22" />
    <link href="libs/remark-css-0.0.1/default.css" rel="stylesheet" />
    <link href="libs/remark-css-0.0.1/default-fonts.css" rel="stylesheet" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# Introduction to BioinformaticsðŸ§¬ðŸ’»
### Martin Skarzynski
### 2019-11-22

---


&lt;style type="text/css"&gt;
.remark-slide-content {
    font-size: 35px;
    padding: 1em 4em 1em 4em;
}
&lt;/style&gt;



# Outline
- What is bioinformatics?
    - NCBI portal, GenBank, PubMed
- Sequence analysis
    - NCBI BLAST
- Multiple sequence alignment
    - Clustal Omega
- Next generation sequencing
    - UCSC genome browser

---

# What is bioinformatics?

&lt;img src="http://omgenomics.com/wp-content/uploads/2017/03/Venn-diagram-2-1024x576.png" height="550" /&gt;


---

# What is bioinformatics?
- DEFINITION
    - Inter-disciplinary field that combines biology, computer science, mathematics, statistics and engineering
- GOALS
    - Understand biological data
        - DNA, RNA, protein sequences and/or structures
    - Solve biological problems
        - Human disease, physiological traits

---

# Explosion in sequence data
&lt;img src="https://www.genome.gov/sites/default/files/inline-images/costpermb_2017.jpg" height="450" /&gt;

---

# National Center for Biotechnology Information (NCBI)
- Portal to extensive database of resources
    - Analysis tools - BLAST 
    - Sequence data - GenBank, GEO
    - Scientific literature - PubMed
    - Protein structures, Variation, etc

---

# [NCBI website](https://ncbi.nlm.nih.gov)

&lt;img src="https://s3.amazonaws.com/libapps/accounts/49010/images/NCBIHome.png" height="450" /&gt;

---

# [GenBank](https://ncbi.nlm.nih.gov/genbank/)

&lt;img src="https://upload.wikimedia.org/wikipedia/commons/f/fc/Genbank.jpg" height="450" /&gt;

---

# Demo
- Search `gapdh` in GenBank
- Available information
  - Sequence, genomic context, tissue expression, linked articles, etc.

---

# [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/)

&lt;img src="https://www.gfmer.ch/SRH-Course-2015/research-methodology/images/Enter-terms-search-box1.jpg" height="450" /&gt;

---

# Sequence Search
- Problem
  - Search for an unknown (query) sequence from a large database of genomes
- BLAST = Basic Local Alignment Search Tool

---

# BLAST workflow
- Choose appropriate BLAST program 
- Enter query sequence
- Select database to search 
- Run BLAST search 
- Analyze output

---

# [NCBI BLAST](https://blast.ncbi.nlm.nih.gov/)

&lt;img src="https://widdowquinn.github.io/2018-03-06-ibioic/02-sequence_databases/images/01-01_ncbi_landing.png" height="450" /&gt;

---

# Nucleotide BLAST (BLASTN)

&lt;img src="https://guides.nnlm.gov/uploads/images/nuc_blast.png" height="450" /&gt;

---

# Nucleotide BLAST (BLASTN)

&lt;img src="https://1.bp.blogspot.com/-GtM5EGPh1Ks/UD_S4DmT84I/AAAAAAAAhyw/68cTUWXeFpE/s1600/Screen+Shot+2012-08-30+at+1.50.50+PM.png" height="450" /&gt;

---

# Protein BLAST (BLASTP)

&lt;img src="https://s3.amazonaws.com/libapps/accounts/5290/images/NCBI_LibGuide_12.jpg" height="450" /&gt;

---

# BLAST output
- Job summary
- Graphic summary

&lt;img src="https://guides.nnlm.gov/uploads/images/results_graph.png" height="450" /&gt;

---

# BLAST output
- Color-coded based on scores 
  - Mouse-over for more information


&lt;img src="https://imgix.albert.io/user-assets/jawagoner36/d750ffa6-63ab-4da4-8ed6-b800cf1230f8-BLAST%20results%20picture%20small.png?ixjsv=2.2.4&amp;rect=23%2C0%2C1098%2C458&amp;w=1" height="350" /&gt;

---

# BLAST output

- Ranked alignments with scores
- Higher scores = better alignment 
- Lower E-values = lower probability that alignment
was random chance

&lt;img src="https://sjc2.discourse-cdn.com/standard10/uploads/mothur/original/1X/345a0d44652af3f67426b5a9ec88e7a47396d91c.png" height="300" /&gt;


---

# BLAST output
- Detailed alignments
- Visual representation of alignment 
- Can include gaps or mismatches

&lt;img src="https://www.ncbi.nlm.nih.gov/books/NBK62051/bin/blast_glossary-Image004.jpg" height="300" /&gt;

---

# Demo
- https://digitalworldbiology.com/BLAST/sequences
- Copy and paste sequences into BLAST
  - Examine results
  
---

# [Galaxy](https://galaxyproject.github.io/)

&lt;img src="https://galaxyproject.github.io/static/galaxyproject.jpg" height="300" /&gt;

---

# [Galaxy BLAST](https://galaxyproject.github.io/training-material/topics/genome-annotation/tutorials/genome-annotation/tutorial.html#similarity-searches-blast)

&lt;img src="https://galaxyproject.github.io/training-material/topics/genome-annotation/images/blastP.png" height="450" /&gt;

---

# Group Activity

1. Find the sequence of a protein of interest 
1. BLAST the sequence to make sure it is correct
1. Find the sequence's molecular weight (in kilodatons)

---


# Multiple sequence alignment
- Alignment of three or more sequences to infer sequence homology and study evolutionary relationships
  - Protein, DNA or RNA
- CLUSTAL
  - **CLUST**er Analysis of Pairwise **AL**ignments

---

# CLUSTAL workflow
- Input sequences or upload file 
- Specify parameters
- Run CLUSTAL 
- Analyze results

---

# [Clustal Omega](http://www.ebi.ac.uk/Tools/msa/clustalo/)

&lt;img src="https://cdn-images-1.medium.com/max/2400/1*BtZ6GNh_768JAvtCQZE7GQ.png" height="450" /&gt;

---

# Clustal Omega Alignment

&lt;img src="https://i.ytimg.com/vi/IM2CTI2rz0k/maxresdefault.jpg" height="450" /&gt;

---

# Clustal Omega Alignment

| Symbol | Meaning                   |
|--------|---------------------------|
| *      | Fully conserved residue   |
| :      | Strongly related residues |
| .      | Weakly related residues   |

---

# Clustal Omega Alignment

| Color   | Residues | Properties                                      |
|---------|----------|-------------------------------------------------|
| Red     | AVFPMILW | Small, hydrophobic, aromatic -Y               |
| Blue    | DE       | Acidic                                          |
| Magenta | RK       | Basic                                           |
| Green   | STYHCNGQ | Hydroxyl, sulfhydryl, amine, G                  |
| Grey    | Others   | Unusual amino acids                             |

---

# Clustal Omega Output
- Phylogenetic tree

&lt;img src="https://cdn.kastatic.org/ka-perseus-images/6a7912e4f113f2e0dde68a8292aa06170e1cff00.png" height="400" /&gt;

---

# Next-generation sequencing (NGS)
- Quantum leap of DNA sequencing technology 
- Ability to sequence whole genomes fast and cheap 
- Large repository of public data
- ENCODE project
- Tools to browse genomes and view data
- UCSC genome browser

---

# [ENCODE project](https://encodeproject.org/)
- Encyclopedia of DNA elements
- https://www.encodeproject.org/
- Launched in 2003 to identify all functional elements in human genome
- Extended to mouse, worm, fly
- Massive consortium led by NHGRI

&lt;img src="https://pbs.twimg.com/profile_images/2019909630/ENCODE_logo_NHGRI.JPG" height="200" /&gt;
---

# [ENCODE project](https://encodeproject.org/): Assays

&lt;img src="https://www.encodeproject.org/static/img/classic-image.jpg" height="400" /&gt;

---

# [UCSC Genome Browser](https://genome.ucsc.edu/)
- Large public database of genome data
- Linked to many public
repositories, e.g. ENCODE
- Easy to use 
- Customizable view 
-  Can upload your own data ("custom tracks")

---

# [UCSC Genome Browser](https://genome.ucsc.edu/)

&lt;img src="https://ucscgenomics.soe.ucsc.edu/wp-content/uploads/2016/05/Untitled-6.png" height="450" /&gt;

---

# [UCSC Genome Browser](https://genome.ucsc.edu/)

&lt;img src="https://i.ytimg.com/vi/wmmqYL4jdAo/maxresdefault.jpg" height="450" /&gt;

---
# Annotation tracks

&lt;img src="https://upload.wikimedia.org/wikipedia/commons/c/c4/UCSC_Tracks.jpg" height="450" /&gt;

---

# ENCODE @ UCSC
- Separate website with ENCODE data also available https://genome.ucsc.edu/ENCODE/
- Search human data
- Search mouse data
- Search full database for tracks to display

---

# Recap
- What is bioinformatics?
- NCBI portal - GenBank, Pubmed
- Sequence analysis
- NCBI BLAST
- Galaxy
- Multiple sequence alignment
- Clustal Omega
- Next-generation sequencing (NGS)
- UCSC Genome Browser - ENCODE project

---

# Questions?

---

# Demo files
- BLAST
- https://digitalworldbiology.com/BLAST/sequences
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false,
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
